Revise the "Add ScalarDB to Your Build" doc (#1028)
scalar-labs/scalardb: docs/add-scalardb-to-your-build.md

> # Add ScalarDB to Your Build

Author: I wonder, should the title be something like the following instead?

```suggestion
# Add the ScalarDB Build to Your Application
```

Reviewer: We normally call such a library "dependency". So, IMO, "Add dependency on ScalarDB to your build configuration"  would be more precise, but it is verbose...

Reviewer 2: Can we consider ScalarDB as a build as it is a finished product, not an unfinished one?
Do you think the original title is too vague?

Author: @Torch3333 Thanks for that link! I realize now that my previous suggestion was incorrect.

I feel the original title is a little vague, but maybe just adding `Application` helps clarify it. What do you think?

```suggestion
# Add ScalarDB to Your Application Build
```

Reviewer 3: I think it's fine as is since another major project also says the same?
(or, we took it from the guava project maybe.)
https://github.com/google/guava#adding-guava-to-your-build

Author: Alright. I'm OK with it as is.

The ScalarDB library is available on the [Maven Central Repository](https://mvnrepository.com/artifact/com.scalar-labs/scalardb). You can add the library as a build dependency to your application by using Gradle or Maven.

## Configure your application based on your build tool

Select your build tool, and follow the instructions to add the build dependency for ScalarDB to your application.

<div id="tabset-1">
<div class="tab">
  <button class="tablinks" onclick="openTab(event, 'Gradle', 'tabset-1')" id="defaultOpen-1">Gradle</button>
  <button class="tablinks" onclick="openTab(event, 'Maven', 'tabset-1')">Maven</button>
</div>

<div id="Gradle" class="tabcontent" markdown="1">

To add the build dependency for ScalarDB by using Gradle, add the following to `build.gradle` in your application, replacing `<VERSION>` with the version of ScalarDB that you want to use:

```gradle
dependencies {
  implementation 'com.scalar-labs:scalardb:<VERSION>'
}
```
</div>
<div id="Maven" class="tabcontent" markdown="1">

> To add the build dependency for ScalarDB by using Maven, add the following to `~/.m2/settings.xml` in your application, replacing `<VERSION>` with the version of ScalarDB that you want to use:

Reviewer 4: ```suggestion
To add the build dependency for ScalarDB by using Maven, add the following to `pom.xml` in your application, replacing `<VERSION>` with the version of ScalarDB that you want to use:
```
